main.cpp

```diff
@@ -222,43 +244,178 @@ IntType& IntType::divide(int i)
     return *this;
 }
 
+// ===== Point Constructors ===== //
+
+Point::Point(const FloatType& f) : x(f), y(f) {}
+Point::Point(const DoubleType& d) : x(static_cast<float>(d)), y(static_cast<float>(d)) {}
+Point::Point(const IntType& i) : x(static_cast<float>(i)), y(static_cast<float>(i)) {}
+
+// ===== Point Member Function Implementations ===== //
+
+void Point::toString()
+{
+    std::cout << "x: " << x << " y: " << y << std::endl;
+}
+
+Point& Point::multiply(FloatType& f)
+{
+    x *= f;
+    y *= f;
+    return *this;
+}
+
+Point& Point::multiply(DoubleType& d)
+{
+    x *= static_cast<float>(d);
+    y *= static_cast<float>(d);
+    return *this;
+}
+
+Point& Point::multiply(IntType& i)
+{
+    x *= static_cast<float>(i);
+    y *= static_cast<float>(i);
+    return *this;
+}
```

Comment:
D.R.Y. 
- [ ] call `multiply(float)` internally.   Be sure to note what `multiply(float)` returns